src/core/hle/kernel/vm_manager.h

```diff
@@ -450,6 +451,34 @@ class VMManager final {
     ///
     ResultVal<VAddr> SetHeapSize(u64 size);
 
+    /// Maps memory at a given address.
+    ///
+    /// @param addr The virtual address to map memory at.
+    /// @param size The amount of memory to map.
+    ///
+    /// @note The destination address must lie within the Map region.
+    ///
+    /// @note This function requires SystemResourceSize is non-zero,
```

Comment:
```suggestion
    /// @note This function requires that SystemResourceSize be non-zero,
```

Ditto below for UnmapPhysicalMemory too.